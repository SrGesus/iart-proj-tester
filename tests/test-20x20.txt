VC	LH	LH	LV	LH	LH	LV	LV	LH	VE	FB	BC	BC	FD	VE	LH	FC	VB	LV	VD
LH	VD	LH	VD	FB	FC	FD	BD	VC	VE	VD	FD	BC	LH	BC	VC	BB	BB	FD	FE
FD	FE	FE	BD	BC	BC	VB	FC	VC	LH	BD	BB	BD	VE	BE	BC	VB	BD	LH	FC
FD	FC	VC	FD	VC	BB	BB	FE	LH	LH	LH	BE	VE	FE	FD	VB	FE	VE	LV	FC
VC	LH	BD	BB	LV	FB	BE	BB	LV	BC	LH	FC	BE	BD	BE	LH	BC	LH	BC	VE
FC	FC	FD	BE	VD	FD	BE	BB	FB	BE	FE	VC	BC	FC	VC	FE	BB	VB	LH	FD
VB	BB	VD	FB	FC	LH	BB	FC	VC	VD	FB	FE	BE	LV	BC	VC	FE	FD	VB	FC
FD	LH	BD	BC	FE	BE	BD	VC	BC	FD	BC	BB	BB	VE	LH	BE	BD	LV	BB	FE
VC	LH	BE	BC	LV	VE	LH	LV	LH	FC	LV	BB	VB	VD	LH	LV	VD	VB	BC	FE
LV	FD	VB	FD	BB	BC	BC	FB	LH	FD	BE	BD	FE	FD	BB	VB	VC	LH	BC	VC
LH	VC	LH	VB	FB	LV	BC	VE	BC	FB	LV	BE	FE	FD	VC	FB	BE	FB	FE	LV
FD	FB	VE	BC	BC	BE	FE	FD	BD	VE	LH	VB	FD	VB	BE	BE	VB	VD	FD	LV
VB	BC	BB	FC	LH	BE	VE	FB	BD	LH	BD	FD	VD	BC	LH	FC	VC	VC	FC	BC
LV	BD	VC	FC	BE	LV	FB	FD	LH	LH	BB	VE	FC	LH	BD	BD	BD	VD	BB	BB
FB	LH	LV	FE	VE	BB	BB	VC	FE	BC	BE	FE	VB	BD	LV	FD	FC	FC	LH	LV
FC	BC	VE	VE	FE	LH	BC	BB	FB	FD	BB	FC	LH	LV	BB	VC	VD	BB	BE	LH
VC	BE	VD	FD	BB	BC	LV	VD	BC	FE	LV	VE	BC	LV	LH	LH	LH	LH	FD	LV
LH	VE	VB	VC	BC	LH	VB	VE	VC	FB	FC	LV	LV	FC	LV	FD	FB	BE	FD	FE
LV	FC	VC	VC	LV	LH	VD	BC	BC	FB	VD	BD	FB	FE	VE	FB	BB	BC	BE	VD
VD	FC	FD	FE	VB	FD	VD	FE	FE	FD	VC	VE	LV	FC	FB	LH	VB	FB	VB	FC
