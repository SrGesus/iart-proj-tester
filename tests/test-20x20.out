VB	LH	LH	LH	LH	LH	LH	LH	LH	VE	FD	BB	BB	FE	VB	LH	FE	VB	LH	VE
LV	VB	LH	VE	FB	FB	FD	BB	VE	VD	VE	FC	BD	LH	BE	VB	BB	BC	FE	FC
FC	FC	FD	BC	BC	BC	VE	FC	VD	LH	BC	BB	BC	VE	BD	BE	VD	BB	LH	FE
FB	FD	VE	FB	VB	BB	BE	FD	LH	LH	LH	BC	VE	FC	FC	VD	FE	VD	LH	FE
VD	LH	BC	BE	LV	FC	BD	BB	LH	BB	LH	FE	BD	BB	BB	LH	BB	LH	BB	VE
FB	FB	FD	BC	VC	FD	BE	BD	FE	BD	FE	VB	BE	FC	VD	FE	BD	VE	LV	FC
VD	BC	VE	FB	FD	LH	BE	FC	VB	VC	FB	FC	BD	LH	BB	VE	FC	FC	VD	FE
FD	LH	BC	BE	FD	BB	BE	VB	BE	FD	BC	BB	BC	VE	LV	BD	BB	LH	BB	FE
VB	LH	BB	BC	LH	VC	LV	LV	LV	FD	LH	BE	VB	VC	LV	LV	VD	VE	BD	FE
LV	FD	VC	FD	BB	BB	BE	FC	LV	FD	BB	BE	FC	FD	BE	VD	VE	LV	BD	VE
LV	VB	LH	VE	FC	LV	BD	VE	BD	FE	LV	BD	FE	FD	VC	FD	BE	FC	FC	LV
FC	FC	VB	BC	BB	BE	FC	FC	BD	VE	LV	VD	FE	VB	BB	BB	VC	VB	FE	LV
VB	BB	BC	FE	LV	BD	VE	FD	BE	LV	BD	FE	VB	BE	LV	FC	VB	VC	FD	BE
LV	BD	VE	FD	BE	LV	FC	FB	LV	LV	BD	VE	FC	LV	BD	BB	BE	VB	BB	BE
FC	LV	LV	FD	VC	BD	BB	VC	FC	BD	BE	FC	VB	BE	LV	FC	FC	FC	LV	LV
FD	BE	VD	VE	FB	LV	BD	BB	FE	FC	BD	FE	LV	LV	BD	VE	VB	BB	BE	LV
VB	BC	VE	FC	BD	BE	LV	VD	BB	FE	LV	VB	BE	LV	LV	LV	LV	LV	FC	LV
LV	VB	VC	VB	BE	LV	VD	VE	VD	FE	FC	LV	LV	FC	LV	FC	FC	BD	FE	FC
LV	FC	VB	VC	LV	LV	VB	BC	BB	FE	VB	BE	FC	FD	VC	FD	BB	BC	BB	VE
VD	FE	FC	FD	VC	FC	VD	FE	FC	FD	VC	VD	LH	FE	FD	LH	VC	FD	VC	FC
